app/assets/javascripts/live_search.js

```diff
@@ -115,6 +115,14 @@
       this.$suggestionsBlock.setAttribute('data-search-query', this.currentKeywords())
     }
     if (GOVUK.Ecommerce) { GOVUK.Ecommerce.start() }
+
+    if (GOVUK.Modules) {
```

Comment:
If the intent of this check is to make sure that the GA4 ecommerce tracker exists before running, would it be better to change it to `if (GOVUK.Modules.Ga4EnhancedEcommerceTracker) {`?

Reply:
Good point! I've made this change in the new implementation.